docs/spec/sanitations.md

```diff
@@ -1,24 +1,40 @@
-_Author_:  <!-- TODO: Add author name --> \
-_Created_: <!-- TODO: Add date --> \
-_Updated_: <!-- TODO: Add date --> \
+_Author_: @AnsarMahir \
+_Created_: 10/25/2024 \
+_Updated_: 10/25/2024 \
 _Edition_: Swan Lake
 
 # Sanitation for OpenAPI specification
 
 This document records the sanitation done on top of the official OpenAPI specification from Jira. 
-The OpenAPI specification is obtained from (TODO: Add source link).
+The OpenAPI specification is obtained from ([Atlassian official website](https://developer.atlassian.com/cloud/jira/platform/swagger-v3.v3.json)).
 These changes are done in order to improve the overall usability, and as workarounds for some known language limitations.
 
 [//]: # (TODO: Add sanitation details)
-1. 
-2. 
-3. 
+1. Removed the request body and the request body schema of the operation getIssueLimitReport (GET) ( /rest/api/3/issue/limit/report)
```

Comment:
Let's say we removed these because they were marked as deprecated. You can point to the official web page which gives information on deprecated fields.
https://developer.atlassian.com/cloud/jira/platform/deprecation-notice-user-privacy-api-migration-guide/

Reply:
Please check the above comment and fix.